kuma/wiki/management/commands/submit_deleted_documents.py

```diff
@@ -0,0 +1,96 @@
+from datetime import timedelta
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db import models
+from django.utils import timezone
+
+from kuma.spam.akismet import Akismet, AkismetError
+from kuma.wiki.forms import revision_akismet_parameters
+from kuma.wiki.models import (Document, DocumentDeletionLog,
+                              RevisionAkismetSubmission)
+
+
+class Command(BaseCommand):
+    help = "Seed Akismet spam with documents that were deleted because they were spam"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('username', type=str)
+        parser.add_argument('-n', '--dry-run',
+                            action='store_true', dest='dry_run', default=False,
+                            help="Do everything except actually submit.")
+        parser.add_argument('-d', '--days',
+                            action='store_true', dest='days', default=365,
+                            help="Number of days of deletion logs to inspect.")
+
+    def handle(self, *args, **options):
+        dry_run = options['dry_run']
+
+        ttl = timezone.now() - timedelta(days=options['days'])
+        # first get the last year's deleted document logs
+        logged_deletions = DocumentDeletionLog.objects.filter(
+            timestamp__lte=ttl,
+        ).filter(
+            models.Q(reason__icontains='spam') |
+            models.Q(reason__icontains='junk')
+        )
+        count = logged_deletions.count()
+        self.stdout.write('Checking %s deleted document logs' % count)
+
+        sender = get_user_model().objects.get(username=options['username'])
+        self.stdout.write(u'Submitting spam to Akismet as user %s' % sender)
+
+        akismet = Akismet()
+
+        if not akismet.ready:
+            raise CommandError('Akismet client is not ready')
+
+        for i, logged_deletion in enumerate(logged_deletions.iterator(), 1):
+            self.stdout.write('%d/%d: ' % (i, count), ending='')
```

Comment:
Seems silly to add a requirement just for this bug tqdm or something similar is pretty super. That's something I meant to redo with Input, but never got around to it.